Dockerfile.tmpl

```diff
@@ -167,8 +177,9 @@ RUN pip install pysal && \
     # Use `conda install -c h2oai h2o` once Python 3.7 version is released to conda.
     apt-get install -y default-jre-headless && \
     pip install -f https://h2o-release.s3.amazonaws.com/h2o/latest_stable_Py.html h2o && \
-    pip install tensorflow-gcs-config==2.6.0 && \
-    pip install tensorflow-addons==0.14.0 && \
+    pip install tensorflow-gcs-config==${TENSORFLOW_VERSION} && \
+    # TODO(b/207851560) Upgrade to 0.17.1 once the base image with TensorFlow 2.9.1 is out.
+    pip install tensorflow-addons==0.17.0 && \
     pip install tensorflow_decision_forests==0.2.0 && \
```

Comment:
With TensorFlow 2.9.1, you can use `tensorflow_decision_forests==0.2.7`: https://github.com/tensorflow/decision-forests/blob/af7633447acc2146ddbf0547165098ff592711e6/configure/setup.py#L31